account_multicurrency_revaluation/model/account.py

```diff
@@ -82,10 +167,12 @@ def _revaluation_query(self, revaluation_date):
         aml.currency_id,
         aml.debit,
         aml.credit,
-        aml.amount_currency
+        aml.amount_currency,
+        aml.id as origin_aml_id
     FROM """
             + tables
             + """
+    LEFT JOIN account_move am ON aml.move_id = am.id
```

Comment:
```suggestion
```
It is now added by the tables argument.

Reply:
Nevermind, before this commit https://github.com/odoo/odoo/commit/c2a72c268e6a28d7448b6dbf96fed4e36d24b3fe Odoo was using the move_id.state which caused the method _where_calc to add a left join with the account_move table. After that, the left join is not added anymore.

So we need to add it manually.